modules/virt-define-http-liveness-probe.adoc

```diff
@@ -4,74 +4,41 @@
-+
---
-* You configure a probe using `spec.livenessProbe.httpGet`, which queries port `1500` of the virtual machine instance, after an initial delay of `120` seconds.
-* The virtual machine instance installs and runs a minimal HTTP server
-on port `1500` using `cloud-init`.
---
-+
-[NOTE]
-====
-The `timeoutSeconds` value must be lower than the `periodSeconds` value. The `timeoutSeconds` default value is `1`. The `periodSeconds` default value is `10`.
-====
+. Include details of the HTTP liveness probe in the VMI configuration file.
 +
+
+.Sample liveness probe with an HTTP GET test
 [source,yaml]
 ----
-apiVersion: kubevirt.io/v1alpha3
-kind: VirtualMachineInstance
-metadata:
-  labels:
-    special: vmi-fedora
-  name: vmi-fedora
 spec:
-  domain:
-    devices:
-      disks:
-      - disk:
-          bus: virtio
-        name: containerdisk
-      - disk:
-          bus: virtio
-        name: cloudinitdisk
-    resources:
-      requests:
-        memory: 1024M
   livenessProbe:
-    initialDelaySeconds: 120
-    periodSeconds: 20
-    httpGet:
-      port: 1500
-    timeoutSeconds: 10
-  terminationGracePeriodSeconds: 0
-  volumes:
-  - name: containerdisk
-    registryDisk:
-      image: kubevirt/fedora-cloud-registry-disk-demo
-  - cloudInitNoCloud:
-      userData: |-
-        #cloud-config
-        password: fedora
-        chpasswd: { expire: False }
-        bootcmd:
-          - setenforce 0
-          - dnf install -y nmap-ncat
-          - systemd-run --unit=httpserver nc -klp 1500 -e '/usr/bin/echo -e HTTP/1.1 200 OK\\n\\nHello World!'
-    name: cloudinitdisk
+    initialDelaySeconds: 120 <1>
+    periodSeconds: 20 <2>
+    httpGet: <3>
+      port: 1500 <4>
+      path: /healthz <5>
+        httpHeaders:
+        - name: Custom-Header
+          value: Awesome
+    timeoutSeconds: 10 <6>
 ----
-+
-. Create the virtual machine instance by running the following command:
+<1> The time, in seconds, after the VMI starts before the liveness probe is initiated.
+<2> The delay, in seconds, between performing probes. The default delay is 10 seconds. This value must be greater than `timeoutSeconds`.
+<3> The HTTP GET request to perform to connect to the VMI.
+<4> The port of the VMI that the probe queries. In the above example, the probe queries port 1500. The VMI installs and runs a minimal HTTP server on port 1500 via cloud-init.
+<5> The path to access on the HTTP server. In the above example, if the handler for the server's `/healthz` path returns a success code, the VMI is considered to be healthy. If the handler returns a failure code, the VMI is deleted and a new instance is created.
+<6> The number of seconds of inactivity after which the probe times out and the VMI is assumed to have failed. The default value is 1. This value must be lower than `periodSeconds`.
+
+. Create the VMI by running the following command:
 +
 [source,terminal]
 ----
-$ oc create -f <file name>.yaml
+$ oc create -f <vmi-config>.yaml
```

Comment:
The other two modules use `<file_name>.yaml`; I think it would be best if they were all the same. Outside of that, this should use an underscore instead of a hyphen.

Reply:
Fixed